ddtrace/contrib/graphql/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""
+This integration instruments ``graphql-core`` queries. Version 2.0 and above are supported.
+
+
+Enabling
+~~~~~~~~
+
+The graphql integration is enabled automatically when using
+:ref:`ddtrace-run <ddtracerun>` or :func:`patch_all() <ddtrace.patch_all>`.
+
+Or use :func:`patch() <ddtrace.patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(graphql=True)
+    import graphql
+    ...
+
+Global Configuration
+~~~~~~~~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.graphql["service"]
+
+   The service name reported by default for graphql instances.
+
+   This option can also be set with the ``DD_SERVICE`` environment
+   variable.
+
+   Default: ``"graphql"``
+
+.. py:data:: ddtrace.config.graphql["resolvers_enabled"]
+
+   To enable ``graphql.resolve`` spans set ``DD_TRACE_GRAPHQL_RESOLVERS_ENABLED`` to True
```

Comment:
Maybe give just a hint of context for why this feature should or should not be enabled (warning if too many spans).